data_cleaning/ice_velocity.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<p>When I tested this for <code>vv</code> and other variables/subsets I get <code>False</code> even though the values appear to be the same. I was wondering if it could be because the variable names and the coordinates are different between the two?   </p>

Reply:
<p>Are the values floats? Xarray docs for <code>.equals()</code> don't say how floats are compared, which could lead to <code>False</code>... </p>

Reply:
In that case we should switch to [xr.testing.assert_allclose](https://docs.xarray.dev/en/stable/generated/xarray.testing.assert_allclose.html). For some reason we don't have `DataArray.isclose`

Reply:
Using `xr.testing.assert_allclose()` raises an error, I think because the coords are different between two objects? using `np.testing.assert_allclose()` on the `.data` arrays seems to work. Do you think this is worth keeping in /explaining? 
<img width="676" alt="Screen Shot 2022-07-08 at 2 21 37 PM" src="https://user-images.githubusercontent.com/55526386/178065010-86b955fa-8e3f-4c24-aff7-c04a38e9a8e8.png">

Reply:
AH right, call it with `.sel(time=1996, drop=True)`